src/gui/newSong.cpp

```diff
@@ -167,7 +159,8 @@ void FurnaceGUI::drawNewSong() {
     selEnd=SelectionPoint();
     cursor=SelectionPoint();
     updateWindowTitle();
-    firstFrame = true;
     ImGui::CloseCurrentPopup();
   }
+
+  newSongFirstFrame=false;
```

Comment:
I still had to add a line setting this at the end of the function (so it's only true on the first frame). Is this okay?

Reply:
Looks good.